src/connection.rs

```diff
@@ -432,6 +432,14 @@ impl<T: AsyncRead + AsyncWrite + Unpin> Sender<T> {
         self.send_frame(&mut header, &mut Storage::Shared(data.as_ref().as_bytes())).await
     }
 
+    /// Send a text value over the websocket connection.
+    ///
+    /// This method performs one fewer copy than [`Sender::send_text`].
```

Comment:
maybe?!

```suggestion
    /// This method performs less copy than [`Sender::send_text`] if the data is already a String.
```

Reply:
I had to think about this now, should probably be "one copy fewer" (it's definitely "fewer", not "less", since you can count the copies).

Reply:
"one copy fewer" sounds alright to me but I'm not native english speak as you know :)